_i18n/en/documentation/general/f-droid.md

```diff
@@ -1,10 +1,18 @@
-After a new version of AntennaPod gets released, we often are asked when it will be available on F-Droid. We can't give an exact answer — it usually takes a few days.
+Did you see that there's a new AntennaPod release and wonder:
+* _why isn't the latest version not on F-Droid yet?_
+* _when will the new release be available on F-Droid?_
 
-Getting a new version on F-Droid involves the following steps:
+If the new version is in beta, it is _only_ available through Google Play ((here is why)[/documentation/general/beta]). When the new version is ready for the public, we upload it to Google Play and tag the release on GitHub. F-Droid then has to find out about this release, build it and make it available. All these steps may take some time.
```

Comment:
```suggestion
If the new version is in beta, it is _only_ available through Google Play ((here is why)[/documentation/general/beta]). When the new version is ready for the public, we upload it to Google Play. A few days later when we are sure that there are no critical bugs, we tag the release on GitHub for F-Droid to find. F-Droid then has to find out about this release, build it and make it available. All these steps may take some time.
```

Reply:
When writing the text I was hesitating to include what you wrote. I then decided against it because it adds a level of detail (extra days between first Google upload to small percentage of users and GitHub tag) that is not relevant for the F-Droid users.
Would you be OK with leaving as is, not to make the text longer (I already feel I've been too wordy).

Reply:
Hmm. Not sure adding this would've prevented the user from creating a ticket. Arguably they didn't even look at our documentation. Also, "When the new version is ready for the public" and "when we are sure that there are no critical bugs" are a bit contradictory: why do we see something as 'ready for the public' if we're not sure yet that there are no critical bugs...

Reply:
On Google Play, we are a lot more flexible (can stop roll-out, quickly upload an update, etc). So being extra sure that there are no critical bugs does make sense, even if it is already ready for the public. I do have to admit that it sounds a bit weird here, though. Not sure how to best say this.